src/pybind/mgr/orchestrator/module.py

```diff
@@ -663,6 +674,8 @@ def ukn(s: Optional[str]) -> str:
                     status,
                     nice_delta(now, s.last_refresh, ' ago'),
                     nice_delta(now, s.created),
+                    nice_bytes(s.memory_usage),
+                    nice_bytes(s.memory_request) + ' (' + nice_bytes(s.memory_limit) + ')',
```

Comment:
Paul added a `wide` flage for `device ls` due to the table being too wide to fit on usual screens:

https://github.com/ceph/ceph/blob/316b4ad614a1f662bcf6a4a336b1deb592aeea93/src/pybind/mgr/orchestrator/module.py#L455-L465

wdyt?